test_autoarray/inversion/inversion/test_factory.py

```diff
@@ -124,6 +124,63 @@ def test__inversion_imaging__via_mapper(
     assert inversion.mapped_reconstructed_image == pytest.approx(np.ones(9), 1.0e-4)
 
 
+def test__inversion_imaging__via_regularizations(

```

Comment:
The tests in `test_autoarray.inversion.inversion.test_factory` test many combinations of mappers and regularizations. By asserting on `inversion.log_det_curvature_reg_matrix_term` we can be sure that any change in the code which breaks the internal calculcation is flagged.

To compute the values in this test I have previous assumed the code is working and hard-coded them in. This is not great practise, but important for having tests which will catch errors.

I have put tests on the split regularizations so I know when I edit them next I cant break them.